backend/python/transformers/requirements.txt

```diff
@@ -3,4 +3,7 @@ transformers
 grpcio==1.64.0
 protobuf
 torch
-certifi
\ No newline at end of file
+certifi
+intel-extension-for-transformers
```

Comment:
looks something should go in https://github.com/mudler/LocalAI/blob/master/backend/python/transformers/requirements-intel.txt ? or is this for some other reason?

Reply:
the package belongs there but requirements-intel has `--extra-index-url https://pytorch-extension.intel.com/release-whl/stable/xpu/us/` and intel-extension-for-transformers is not in that repo. I had error during installation putting it there.